incubator/virtualcluster/pkg/vn-agent/server/route.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/util/proxy"
+	"k8s.io/klog"
+)
+
+// InstallHandlers set router and handlers.
+func (s *Server) InstallHandlers() {
+	ws := new(restful.WebService)
+	ws.
+		Path("/pods").
+		Produces(restful.MIME_JSON)
+	ws.Route(ws.GET("").
+		To(s.proxy).
+		Operation("getPods"))
+	s.restfulCont.Add(ws)
+
+	ws = new(restful.WebService)
+	ws.
+		Path("/run")
+	ws.Route(ws.POST("/{podNamespace}/{podID}/{containerName}").
+		To(s.proxy).
+		Operation("getRun"))
+	ws.Route(ws.POST("/{podNamespace}/{podID}/{uid}/{containerName}").
+		To(s.proxy).
+		Operation("getRun"))
+	s.restfulCont.Add(ws)
+
+	ws = new(restful.WebService)
+	ws.
+		Path("/logs/")
+	ws.Route(ws.GET("").
+		To(s.proxy).
+		Operation("getLogs"))
+	ws.Route(ws.GET("/{logpath:*}").
+		To(s.proxy).
+		Operation("getLogs").
+		Param(ws.PathParameter("logpath", "path to the log").DataType("string")))
+	s.restfulCont.Add(ws)
+
+	ws = new(restful.WebService)
+	ws.
+		Path("/containerLogs")
+	ws.Route(ws.GET("/{podNamespace}/{podID}/{containerName}").
+		To(s.proxy).
+		Operation("getContainerLogs"))
+	s.restfulCont.Add(ws)
+}
+
+func (s *Server) proxy(req *restful.Request, resp *restful.Response) {
+	klog.Infof("request %+v", req)
```

Comment:
this logging should not be infof level. maybe at debugging level.

Reply:
changed verbose level here.